packages/react-integration/demo-app-ts/src/components/demos/TabsDemo/TabsHorizontalOverflowDemo.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Tabs, Tab, TabTitleText, Checkbox } from '@patternfly/react-core';
+
+export const TabsHorizontalOverflowDemo: React.FunctionComponent = () => {
+  const [activeTabKey, setActiveTabKey] = React.useState<string | number>(0);
+  const [showTabCount, setShowTabCount] = React.useState(false);
+  const [defaultTitleText, setDefaultTitleText] = React.useState(undefined);
+  const [toggleAriaLabel, setToggleAriaLabel] = React.useState(undefined);
+
+  const handleTabClick = (_event: any, tabIndex: string | number) => {
```

Comment:
nit: Is this event meant to be any? I understand it's not used, but maybe still we could use to correct type.